packages/react/src/patterns/sub-patterns/ContentSection/__stories__/index.scss

```diff
@@ -1,7 +1,30 @@
 @import '../../../../../../styles/scss/patterns/sub-patterns/content-section/content-section';
 
+@mixin content-section-themed-items {
+  color: $text-01;
+  background: $ui-01;
+}
+
 .#{$prefix}--content-section-story .#{$prefix}--content-section__children {
   height: carbon--mini-units(60);
-  background-color: #f4f4f4;
   padding: $carbon--spacing-05;
+  @include themed-items;
```

Comment:
@guilhermelMoraes you'll have to update these includes too...

Reply:
My bad!